packages/synapse-react-client/src/components/dataaccess/AccessRequirementTable.integration.test.tsx

```diff
@@ -61,57 +57,9 @@ function renderComponent(props: AccessRequirementTableProps) {
   )
 }
 
-const onServiceRecievedRequest = jest.fn()
-const onServiceRecievedRequestForNextPage = jest.fn()
-
-const nextPageToken = 'mock-npt'
-
-const mockSearchResultsPage2: AccessRequirementSearchResponse = {
-  results: [
-    {
-      id: '9608424',
-      createdOn: '2017-08-23T18:48:20.892Z',
-      type: ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
-      modifiedOn: '2022-05-20T22:26:44.406Z',
-      name: 'Access Requirement on Page 2',
-      version: '269',
-      relatedProjectIds: [MOCK_PROJECT_ID],
-      reviewerIds: [],
-    },
-  ],
-}
-
 describe('Access Requirement Table tests', () => {
   beforeAll(() => {
     server.listen()
-
-    // Configure MSW
-    server.use(
-      rest.post(
-        `${getEndpoint(
-          BackendDestinationEnum.REPO_ENDPOINT,
-        )}${ACCESS_REQUIREMENT_SEARCH}`,
-
-        async (req, res, ctx) => {
-          onServiceRecievedRequest(req.body)
-          let response
-          if (
-            (req.body as AccessRequirementSearchRequest).nextPageToken ===
-            nextPageToken
-          ) {
-            onServiceRecievedRequestForNextPage()
-            response = mockSearchResultsPage2
-          } else {
-            response = {
-              ...mockSearchResults,
-              nextPageToken: nextPageToken,
-            }
-          }
-
-          return res(ctx.status(200), ctx.json(response))
-        },
-      ),
-    )
```

Comment:
The shared mock implementation was updated to handle pagination, so this can be removed.